teuthology/task/install/util.py

```diff
@@ -125,3 +121,25 @@ def ship_utilities(ctx, config):
                 wait=False,
             ),
         )
+
+utilities_shipped = False
+
+@contextlib.contextmanager
+def ship_utilities(ctx, config):
+    """
+    Write a copy of valgrind.supp to each of the remote sites.  Set executables
+    used by Ceph in /usr/local/bin.  When finished (upon exit of the teuthology
+    run), remove these files.
```

Comment:
I think no need to duplicate docs from _ship_utilitities, it is enough to refer this function and do some more explanation regarding idea of doing it once, for example:
```
Ship utilities during the first call, and skip it in the following ones.
See also `_ship_utilities`.
```

Reply:
Will fix it.